man/tpm2_dictionarylockout.1.md

```diff
@@ -13,11 +13,11 @@
 # DESCRIPTION
 
 **tpm2_dictionarylockout**(1) - setup dictionary-attack-lockout parameters or clear
-dictionary-attack-lockout state, if any passwd option is missing, assume NULL.
+dictionary-attack-lockout state, if any password option is missing, assume NULL.
```

Comment:
Modify dictionary lockout parameters and or clear a TPM lockout state. When password options are not specified the password is assumed to be NULL.

Reply:
@idesai Could point out what should I do? I could not figure out where is the mistake.